### PR TITLE
.NET 6

### PR DESCRIPTION
**Pull request by ErisMik**

### CODE REVIEW

File: binding/dotnet/Porcupine/Porcupine.csproj [hunk 1]
```diff
@@ -72,21 +77,23 @@ Porcupine is:
...
-                build/netcoreapp3.1/lib/beaglebone/libpv_porcupine.so
+                build/netcoreapp3.1/lib/beaglebone/libpv_porcupine.so;
+                build/net6.0/lib/beaglebone/libpv_porcupine.so;
             </PackagePath>
             <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
             <Link>lib\beaglebone\libpv_porcupine.so</Link>
             <Visible>false</Visible>
         </Content>
-		<Content Include="..\..\..\lib\mac\x86_64\libpv_porcupine.dylib">
+		<Content Include="..\..\..\lib\mac\**\*" Exclude="..\..\..\lib\mac\arm11\*">
```
Comment (laves) [inline binding/dotnet/Porcupine/Porcupine.csproj:87]: don't think this arm11 exclude needs to be here

File: binding/dotnet/Porcupine/Utils.cs [hunk 2]
```diff
@@ -59,6 +59,10 @@ public static string PvLibraryPath(string libName)
             {
                 return Path.Combine(Path.GetDirectoryName(Assembly.GetExecutingAssembly().Location), $"lib/{_env}/x86_64/{libName}.dylib");
             }
+            else if (RuntimeInformation.IsOSPlatform(OSPlatform.OSX) && _isArm)
```
Comment (laves) [inline binding/dotnet/Porcupine/Utils.cs:62]: I thought you were also going to check if they were running on dotnet 6.0 when loading M1 lib?

**Review by laves — CHANGES_REQUESTED** [review]

**Comment by laves** [comment]
Just wondering - did you test the package in a .NET 5.0 app?

File: binding/dotnet/Porcupine/Utils.cs (hunk 2 above)
Comment (ErisMik) [reply binding/dotnet/Porcupine/Utils.cs:62]: As far as I can tell it detects the arch of the runtime, not the platform - so we are in the clear :)

**Comment by ErisMik** [comment]
> Just wondering - did you test the package in a .NET 5.0 app?

Just did, it appears to run OK

**Pull request merged**